src/axolotl/utils/models.py

```diff
@@ -613,7 +802,13 @@ def load_model(
         LOG.exception(err)
         raise err
 
-    if isinstance(model, (PeftModel, PeftModelForCausalLM)):
+    qlora_fsdp = (
+        cfg.fsdp
+        and cfg.adapter == "qlora"
+        and model_config.model_type in SUPPORTED_AUTO_WRAP_MODEL_TYPES
+    )
```

Comment:
@winglian this logic is repeated from line 622, perhaps this can be moved up there?

Reply:
Like this https://github.com/OpenAccess-AI-Collective/axolotl/pull/1378/files#r1518165862 then you don't need this line?